### PR TITLE
Update Cromwell Docs for v23

### PR DESCRIPTION
**Pull request by ruchim**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,9 +11,12 @@
   * `cross: (Array[X], Array[Y]) => Array[Pair[X, Y]]` - create every possible pair from the two input arrays and return them all as WDL pairs
   * `transpose: (Array[Array[X]]) => Array[Array[X]]` compute the matrix transpose for a 2D array. Assumes each inner array has the same length.
 * By default, `system.abort-jobs-on-terminate` is false when running `java -jar cromwell.jar server`, and true when running `java -jar cromwell.jar run <wdl> <inputs>`.
-* Enable WDL imports when running in Single Workflow Runner Mode.
+* Enable WDL imports when running in Single Workflow Runner Mode as well as Server Mode
```
Comment (cjllanwarne) [inline CHANGELOG.md]: How about `Enable WDL imports as an additional imports.zip parameter`

File: README.md [hunk 2]
```diff
@@ -1643,11 +1648,29 @@ When running a job on the Config (Shared Filesystem) backend, Cromwell provides
...
+
+In Server Mode, you pass in a zip file using the parameter `wdlDependencies` via the [POST /api/workflows/:version](#post-apiworkflowsversion) endpoint.
+
+
 # Sub Workflows
 
 WDL allows the execution of an entire workflow as a step in a larger workflow (see WDL SPEC for more details), which is what will be referred to as a sub workflow going forward.
 Cromwell supports execution of such workflows. Note that sub workflows can themselves contain sub workflows, etc... There is no limitation as to how deeply workflows can be nested.
 
+# Meta blocks
```
Comment (geoffjentry) [inline README.md]: I don't know that this is an important enough feature that it needs to be in cromwell's readme

File: CHANGELOG.md [hunk 3]
```diff
@@ -11,9 +11,12 @@
...
 * Support for sub workflows
+* Corrected file globbing in JES to be consistent. Additionally, file globbing in JES now uses bash-style glob syntax instead of python style glob syntax
```
Comment (cjllanwarne) [inline CHANGELOG.md]: How about `Corrected file globbing in JES to correctly report all generated files. Additionally.....`

File: CHANGELOG.md [hunk 4]
```diff
@@ -11,9 +11,12 @@
...
 * Support declarations as graph nodes
+* Added the ability to override the default service account that the compute VM is started with via the configuration option `JES.config.genomics.compute-service-account` or through the workflow options parameter `google_compute_service_account`. More details can be found in the README.md
+* Changes to the Single Workflow Runner Mode such that when a workflow completes, Cromwell will exit. Additionally, when restarting in Single Workflow Runner Mode, Cromwell will no longer restart incomplete workflows from a previous session.
```
Comment (cjllanwarne) [inline CHANGELOG.md]: `Changes to the Single Workflow Runner Mode such that when a workflow completes, Cromwell will exit` sounds more like a bug fix than a change?

File: README.md [hunk 5]
```diff
@@ -130,23 +132,29 @@ See the [migration document](MIGRATION.md) for more details.
...
+
+
 $ java -jar cromwell.jar
 java -jar cromwell.jar <action> <parameters>
 
 Actions:
-run <WDL file> [<JSON inputs file> [<JSON workflow options>
-  [<OUTPUT workflow metadata>]]]
+run <WDL file> [<JSON inputs file>] [<JSON workflow options>]
+  [<OUTPUT workflow metadata>] [<Directory of WDL Files>]
```
Comment (cjllanwarne) [inline README.md]: Zip of WDL files

File: README.md [hunk 6]
```diff
@@ -1473,7 +1478,7 @@ When specified, cromwell will run your task within the specified Docker image.
 runtime {
   docker: "ubuntu:latest"
 }
-```
+```f
```
Comment (mcovarr) [inline README.md]: f no

File: README.md [hunk 7]
```diff
@@ -1643,11 +1648,29 @@ When running a job on the Config (Shared Filesystem) backend, Cromwell provides
...
 # Sub Workflows
 
 WDL allows the execution of an entire workflow as a step in a larger workflow (see WDL SPEC for more details), which is what will be referred to as a sub workflow going forward.
 Cromwell supports execution of such workflows. Note that sub workflows can themselves contain sub workflows, etc... There is no limitation as to how deeply workflows can be nested.
 
+# Meta blocks
+
+In order to write notes
+
 ## Execution
```
Comment (Horneth) [inline README.md]: This Execution section as well as everything up to the REST API section are actually sub sections for sub workflows.
The Meta blocks section should be either above Sub Workflows or right before REST API

File: README.md [hunk 8]
```diff
@@ -1643,11 +1648,29 @@ When running a job on the Config (Shared Filesystem) backend, Cromwell provides
...
+
+
 # Sub Workflows
 
 WDL allows the execution of an entire workflow as a step in a larger workflow (see WDL SPEC for more details), which is what will be referred to as a sub workflow going forward.
 Cromwell supports execution of such workflows. Note that sub workflows can themselves contain sub workflows, etc... There is no limitation as to how deeply workflows can be nested.
 
+# Meta blocks
+
+In order to write notes
```
Comment (cjllanwarne) [inline README.md]: Uh, is this sentence unfinished?

File: CHANGELOG.md (hunk 3 above)
Comment (mcovarr) [inline CHANGELOG.md]: consistent how?  I know but the reader may not

**Comment by Horneth** [comment]
👍  after comment is addressed

File: README.md [hunk 9]
```diff
@@ -2115,7 +2138,15 @@ This endpoint accepts a POST request with a `multipart/form-data` encoded body.
 
 * `wdlSource` - *Required* Contains the WDL file to submit for execution.
 * `workflowInputs` - *Optional* JSON file containing the inputs.  A skeleton file can be generated from [wdltool](https://github.com/broadinstitute/wdltool) using the "inputs" subcommand.
+* `workflowInputs_2` - *Optional* JSON file containing the inputs.
+* `workflowInputs_3` - *Optional* JSON file containing the inputs.
+* `workflowInputs_4` - *Optional* JSON file containing the inputs.
+* `workflowInputs_5` - *Optional* JSON file containing the inputs.
```
Comment (cjllanwarne) [inline README.md]: Actually the REST endpoint now accepts an unlimited number of these, but swagger only accepts up to 5

File: CHANGELOG.md (hunk 4 above)
Comment (mcovarr) [inline CHANGELOG.md]: I think that first one is a bugfix and not an intentionally new CHANGELOG-worthy behavior.

File: README.md (hunk 6 above)
Comment (cjllanwarne) [inline README.md]: 🏆

File: CHANGELOG.md (hunk 1 above)
Comment (mcovarr) [inline CHANGELOG.md]: Maybe just "Support for WDL imports" since the only two Cromwell modes are now covered?

File: README.md (hunk 5 above)
Comment (mcovarr) [inline README.md]: Has this really changed with every parameter optional and Cromwell just figures it all out somehow?  The old spec was nested from the end to the front.

File: README.md [hunk 10]
```diff
@@ -1643,11 +1648,29 @@ When running a job on the Config (Shared Filesystem) backend, Cromwell provides
         }
       }
 ```
+# Imports
+
+Import statements inside of a WDL file are supported by Cromwell when running in Server mode as well as Single Workflow Runner Mode.
+
+In Single Workflow Runner Mode, you pass in a zip file which includes the WDL files referenced by the import statements. Cromwell requires the zip file to be passed in as a command line argument, as explained by the section [run](#run)
```
Comment (mcovarr) [inline README.md]: periodless sentence

File: README.md (hunk 2 above)
Comment (ruchim) [inline README.md]: I shall remove.

File: README.md (hunk 7 above)
Comment (ruchim) [inline README.md]: meta section exists in WDL readme, so it's been removed from the cromwell readme.

File: CHANGELOG.md (hunk 4 above)
Comment (ruchim) [inline CHANGELOG.md]: changed wording

File: README.md (hunk 5 above)
Comment (ruchim) [inline README.md]: done

File: README.md (hunk 8 above)
Comment (ruchim) [inline README.md]: its removed

File: README.md (hunk 9 above)
Comment (ruchim) [inline README.md]: I'll add that to the notes below this list of inputs to the submit endpoint

File: CHANGELOG.md (hunk 3 above)
Comment (ruchim) [inline CHANGELOG.md]: modified

File: CHANGELOG.md (hunk 1 above)
Comment (ruchim) [inline CHANGELOG.md]: modified

File: README.md (hunk 5 above)
Comment (ruchim) [inline README.md]: Not totally sure what you mean?

**Pull request merged**